public/css/main.css

```
@@ -230,3 +230,11 @@ tr {
 .status-color-dot.none {
   background-color: #2ecc71;
 }
+
+#libLink {
+    text-decoration: none;
+}
+
+#libLink span:hover {
+    text-decoration: underline;
+}
```

```suggestion
}

```
Missing new line at end of file.